jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -102,7 +107,8 @@
           :single_select_allow_blank="true"
         >
         </plugin-inline-select>
-      </div>
+        </div>
+    </div>

```

Comment:
instead of closing the second div, I think the `<div class="section-description">` should just be removed (seems to erroneously have been introduced in #2760)

Reply:
I fixed this in #2772 (so that we can have the fix in main before the PR is ready for merge).  Once that's merged, rabasing on top of main and dealing with the conflict should fix the problem.